front/app/services/homepageSettings.ts

```diff
@@ -26,7 +26,7 @@ export interface IHomepageSettings {
   id: string;
   // To check
   type: string;
-  attributes: IHomepageSettingsAttributes;
+  data: { attributes: IHomepageSettingsAttributes };
```

Comment:
We usually have a Data type then, as you can see in most services files (eg IIdeaData in ideas.ts).

Reply:
Something like this?

```
export interface IHomepageSettings {
  // To check
  id: string;
  // To check
  type: string;
  data: IHomepageSettingsData;
}

export interface IHomepageSettingsData {
  attributes: IHomepageSettingsAttributes;
}
```

Reply:
Yes!